### DISCUSSION TITLE
Wallpapers

### DISCUSSION THREAD

**Discussion by ghost** [siduck/chadwm#180] [General]
@siduck bro where do you get those seggsy wallpapers??

**Comment by siduck** [comment]
@Subhashis2007 too sad that i dont have most of  them now lol, if you want the green leaves/mountain one then lemme know

**Reply by ghost** [reply]
@siduck send if you have :)

**Reply by siduck** [reply]
![stairs](https://github.com/siduck/chadwm/assets/59060246/8bad929e-c90a-4a5d-9749-be88f62057d4)
![forest-stairs](https://github.com/siduck/chadwm/assets/59060246/a83885f5-1bd8-42a7-8f33-aa1323691a60)
![railtrack](https://github.com/siduck/chadwm/assets/59060246/fa1cdd86-c8e7-4463-953d-8da975ca1c53)
![mount](https://github.com/siduck/chadwm/assets/59060246/9c684c98-deb7-452c-91a8-e992be1c0763)
![snow-mountain](https://github.com/siduck/chadwm/assets/59060246/5a452ffb-0010-4419-a179-b740b3e71d7a)

![forest](https://github.com/siduck/chadwm/assets/59060246/bf3029c7-3299-46b9-9da6-e69cb1b19ce5)
![red_cherryblossom](https://github.com/siduck/chadwm/assets/59060246/64432ef8-46e0-42f4-b23e-3a9edf08b6a5)

**Reply by ghost** [reply]
thanks a lot
